Core/EtagStorage.swift

```diff
@@ -31,7 +31,7 @@ public protocol BlockerListETagStorage {
 
 public struct UserDefaultsETagStorage: BlockerListETagStorage {
 
-    private let defaults = UserDefaults(suiteName: "com.duckduckgo.blocker-list.etags")
+    private let defaults = UserDefaults(suiteName: "\(Global.groupIdPrefix).app-configuration")
```

Comment:
Is this a risky transition / migration?  We're moving from using an existing user defaults suite, to using a new one.

Is there anything stored here that will get reset that might cause trouble for users?

Reply:
I don't think there's a risk here. Configuration starts up and fetches all configs at app launch. For an app update this will look like a fresh install for the configs. It will see no etags and just fetch the current versions and store them. I haven't run in to any issues during testing which involved a "migration".